подання/SphericalCat/спливаюче вікно hooli/styles.css

```diff
@@ -0,0 +1,290 @@
+    height: 60px;
+    box-shadow: 0 5px 10px rgba(34, 63, 87, 0.151);
+}
+
+.menu {
+    display: flex;
+    list-style: none;
+    align-items: center;
+    margin: 0 10px 10px 10px;
+
+}
+
+.menu_item {
+    margin-left: 20px;
+}
+
+.check,
+.check_more {
+    position: absolute;
+    appearance: none;
+    -moz-appearance: none;
+    -webkit-appearance: none;
+}
+
+.see_more{
+    cursor: pointer;
+}
+
+.see_more_item {
+    display: block;
+    text-align: center;
+    padding: 5px;
+    background: #33cccc3b;
+    border-radius: 10px;
+    margin-top: 5px;
+    cursor: pointer;
+}
+
+.see_more_item:hover {
+    background: #33cccc81;
+    color: white;
+}
+
+.popup {
+    display: none;
+    position: absolute;
+    right: 5%;
+    overflow-y: scroll;
+    background: white;
+    border: 2px dotted #33cccc6b;
+    border-radius: 20px;
+    padding: 10px;
+    height: 300px;
+}
+
+.popup_two {
+    margin-top: 10px;
+    display: none;
+}
+
+.check:checked~.popup {
+    display: block;
+}
+
+.check_more:checked~.popup_two {
+    display: block;
+}
+
+.check_more:focus-within~.see_more{
+    background: #33cccc81;
+    color: darkslategrey;
+}
+
+.check_more:checked~.see_more .see_more_item {
+    background: white;
+    border: 2px dotted #33cccc6b;
+}
+
+.check_more:checked~.see_more .see_more_item:hover {
+    color: darkslategray;
+    border: 2px dotted rgba(47, 79, 79, 0.411);
+}
+
+.popup::-webkit-scrollbar {
+    opacity: 0;
+    width: 5px;
+
+}
+
+.popup::-webkit-scrollbar-thumb {
+    background: #33cccc2d;
+    border-radius: 14%;
+}
+
+.popup_items {
+    list-style: none;
+    display: grid;
+    grid-template: repeat(3, 65px)/ repeat(3, 65px);
+    justify-items: center;
+    align-items: center;
+    gap: 20px;
+    text-align: center;
+    padding: 0;
+}
+
+.popup_two_items {
+    display: grid;
+    grid-template-columns: repeat(3, 65px);
+    list-style: none;
+    justify-items: center;
+    align-items: center;
+    gap: 20px;
+    text-align: center;
+    padding: 0;
+}
+
+.burger_img {
+    display: block;
+    background-image: url(styles/img/burger.gif);
+    height: 60px;
+    width: 60px;
+    background-size: cover;
+    background-position: center;
+    cursor: pointer;
+}
+
+.burger_check{
+    cursor: pointer;
+}
+
+.check:checked~.burger_check .burger_img {
+    background-image: url(styles/img/burger.png);
+    background-position: center;
+    background-size: 70%;
+    background-repeat: no-repeat;
+}
+
+.check:focus-within~.burger_check .burger_img {
+    background-image: url(styles/img/burger.png);
+    background-position: center;
+    background-size: 90%;
+    background-repeat: no-repeat;
+}
+
+.items_img {
+    width: 40px;
+    height: 40px;
+    background-size: cover;
+    background-position: center;
+    margin: auto;
+}
+
+.items_img:hover {
+    width: 50px;
+    height: 50px;
+}
+
+.calendar {
+    background-image: url(styles/img/calendar.png);
+}
+
+.calendar:hover {
```

Comment:
I think, that it will be better for UX if `:hover` state will be on `a`, for example 
````
.popup_link:hover .items_img {
    width: 50px;
    height: 50px;
}

.popup_link:hover .calendar {
    background-image: url(styles/img/calendar.gif);
}
````
In this case, menu items will be animated when you hover not only block with image, text too